terrasnek/api.py

```diff
@@ -139,48 +139,48 @@ def __init__(self, api_token, url=TFC_SAAS_URL, verify=True, log_level=LOG_LEVEL
-        self.ssh_keys = None
-        self.teams = None
-        self.team_access = None
-        self.team_memberships = None
-        self.team_tokens = None
-        self.users = None
-        self.user_tokens = None
-        self.vars = None
-        self.workspace_vars = None
-        self.workspaces = None
+        self.admin_orgs                 : TFCAdminOrgs = None
+        self.admin_runs                 : TFCAdminRuns = None  
+        self.admin_settings             : TFCAdminSettings = None
+        self.admin_terraform_versions   : TFCAdminTerraformVersions = None
+        self.admin_users                : TFCAdminUsers = None
+        self.admin_workspaces           : TFCAdminWorkspaces = None
+        self.audit_trails               : TFCAuditTrails = None
+        self.ip_ranges                  : TFCIPRanges = None
+        self.orgs                       : TFCOrgs = None
+
+        self.account                    : TFCAccount = None
+        self.applies                    : TFCApplies  = None
+        self.agents                     : TFCAgents = None
+        self.agent_tokens               : TFCAgentTokens = None
+        self.config_versions            : TFCConfigVersions  = None
+        self.cost_estimates             : TFCCostEstimates = None
+        self.oauth_clients              : TFCOAuthClients = None
+        self.oauth_tokens               : TFCOAuthTokens = None
+        self.org_memberships            : TFCOrgMemberships = None
+        self.org_tokens                 : TFCOrgTokens = None
+        self.plans                      : TFCPlans = None
+        self.plan_exports               : TFCPlanExports = None
+        self.policies                   : TFCPolicies = None
+        self.policy_checks              : TFCPolicyChecks = None
+        self.policy_sets                : TFCPolicySets = None
+        self.policy_set_params          : TFCPolicySetParams = None
+        self.notification_configs       : TFCNotificationConfigurations = None
+        self.registry_modules           : TFCRegistryModules = None
+        self.run_triggers               : TFCRunTriggers= None
+        self.runs                       : TFCRuns = None
+        self.state_versions             : TFCStateVersions = None
+        self.state_version_outputs      : TFCStateVersionOutputs = None
+        self.ssh_keys                   : TFCSSHKeys = None
+        self.teams                      : TFCTeams = None
+        self.team_access                : TFCTeamAccess = None
+        self.team_memberships           : TFCTeamMemberships = None
+        self.team_tokens                : TFCTeamTokens = None
+        self.users                      : TFCUsers = None
+        self.user_tokens                : TFCUserTokens = None
+        self.vars                       : TFCVars = None
+        self.workspace_vars             : TFCWorkspaceVars = None
+        self.workspaces                 : TFCWorkspaces = None
```

Comment:
Need to remove all the spaces before the colons. This does not pass the linter test for contribution.